sample-docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  lnd:
```

Comment:
On this service we still missing if lnd is using bitcoind or btcd as config. Maybe you could checkout sample docker-compose from lnd side! 🙌🏼